spec/web_server

```diff
@@ -15,14 +15,18 @@ var express = require('express'),
 var server = express();
 
 server.use(function(req, res, next) {
-  console.log('%s %s %s', req.method, req.url, req.path);
+  if (!process.env.PLAYWRIGHT_TEST) console.log('%s %s %s', req.method, req.url, req.path);
```

Comment:
I notice that you read this `env` variable twice in this file and I wonder if you couldn't perhaps raise it up as something like `const isPlaywright = !!process.env.PLAYWRIGHT_TEST` earlier on, maybe?

Also, is this environment variable name documented anywhere? It's not clear to me whether this is a name that you came up with or perhaps one that we're inheriting from a dependency. 🤔

Reply:
i've documented the env vars a bit better in 9e5c68e